migrations/2_deploy_contracts.js

```diff
@@ -15,8 +15,9 @@ module.exports = function(deployer, network, accounts) {
     DataFeed,
     "nav-service",                    // _name
     useOraclize,                      // _useOraclize
-    "json(http://9afaae62.ngrok.io/api/sandbox).totalPortfolioValueEth", // _queryUrl
+    "https://coinalpha-oracle-staging.herokuapp.com/api/gdax", // _queryUrl
```

Comment:
you can use the production url:
https://coinalpha-oracle.herokuapp.com/api/gdax